tools/modules/system/manage_dtoverlays.sh

```
@@ -64,6 +64,9 @@ function manage_dtoverlays () {
 			grep '^overlays' ${overlayconf} | grep -qw ${overlay} && status=ON
 			# Raspberry Pi
 			grep '^dtoverlay' ${overlayconf} | grep -qw ${overlay} && status=ON
+   			# handle case where overlay_prefix is part of overlay name
+	  		candidate=$(echo $overlay | sed "s/^$overlay_prefix"-"//g")
```

Shellcheck (0.10.0)

[warning] Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?

(SC2140)